Source/WebCore/platform/gtk/PasteboardGtk.cpp

```diff
@@ -350,7 +350,7 @@ Vector<String> Pasteboard::typesSafeForBindings(const String& origin)
 {
     if (m_selectionData) {
         ListHashSet<String> types;
-        if (auto* buffer = m_selectionData->customData()) {
+        if (auto* buffer = m_selectionData->customData().get()) {
```

Comment:
Wny don't we change `auto*` -> `auto&` and use the `RefPtr` instead of getting the pointer?

Reply:
True!